release-builder.sh

```diff
@@ -50,25 +49,152 @@ STATIC_CONTAINER=$(docker ps -a | grep $CONTAINER_NAME |wc -l)
+
+ARG BUILD_CORES=8
+
+# Enable repositories and install dependencies
+RUN /hbb_exe/activate-exec bash -c "dnf install -y epel-release && \
+    dnf config-manager --set-enabled powertools || dnf config-manager --set-enabled crb && \
+    dnf install -y --enablerepo=devel \
+        wget git \
+        gcc-toolset-11-gcc-c++ gcc-toolset-11-binutils gcc-toolset-11-libatomic-devel \
+        lz4 lz4-devel \
+        ncurses-static ncurses-devel \
+        snappy snappy-devel \
+        zlib zlib-devel zlib-static \
+        libasan \
+        python3 python3-pip \
+        ccache \
+        ninja-build \
+        patch \
+        glibc-devel glibc-static \
+        libxml2-devel \
+        autoconf \
+        automake \
+        texinfo \
+        libtool \
+        llvm14-static llvm14-devel && \
+    dnf clean all"
+
+# Install Conan and CMake
+RUN /hbb_exe/activate-exec pip3 install "conan==1.66.0" && \
+    /hbb_exe/activate-exec wget -q https://github.com/Kitware/CMake/releases/download/v3.23.1/cmake-3.23.1-linux-x86_64.tar.gz -O cmake.tar.gz && \
+    mkdir cmake && \
+    tar -xzf cmake.tar.gz --strip-components=1 -C cmake && \
+    rm cmake.tar.gz
+
+# Install Boost 1.86.0
+RUN /hbb_exe/activate-exec bash -c "cd /tmp && \
+    wget -q https://archives.boost.io/release/1.86.0/source/boost_1_86_0.tar.gz -O boost.tar.gz && \
+    mkdir boost && \
+    tar -xzf boost.tar.gz --strip-components=1 -C boost && \
+    cd boost && \
+    ./bootstrap.sh && \
+    ./b2 link=static -j${BUILD_CORES} && \
+    ./b2 install && \
+    cd /tmp && \
+    rm -rf boost boost.tar.gz"
+
+ENV BOOST_ROOT=/usr/local/src/boost_1_86_0
+ENV Boost_LIBRARY_DIRS=/usr/local/lib
+ENV BOOST_INCLUDEDIR=/usr/local/src/boost_1_86_0
+
+ENV CMAKE_EXE_LINKER_FLAGS="-static-libstdc++"
+
+ENV LLVM_DIR=/usr/lib64/llvm14/lib/cmake/llvm
+ENV WasmEdge_LIB=/usr/local/lib64/libwasmedge.a
+
+ENV CC='ccache gcc'
+ENV CXX='ccache g++'
+
+# Install LLD
+RUN /hbb_exe/activate-exec bash -c "source /opt/rh/gcc-toolset-11/enable && \
+    cd /tmp && \
+    wget -q https://github.com/llvm/llvm-project/releases/download/llvmorg-14.0.3/lld-14.0.3.src.tar.xz && \
+    wget -q https://github.com/llvm/llvm-project/releases/download/llvmorg-14.0.3/libunwind-14.0.3.src.tar.xz && \
+    tar -xf lld-14.0.3.src.tar.xz && \
+    tar -xf libunwind-14.0.3.src.tar.xz && \
+    cp -r libunwind-14.0.3.src/include libunwind-14.0.3.src/src lld-14.0.3.src/ && \
+    cd lld-14.0.3.src && \
+    mkdir -p build && cd build && \
+    cmake .. \
+        -DLLVM_LIBRARY_DIR=/usr/lib64/llvm14/lib/ \
+        -DCMAKE_INSTALL_PREFIX=/usr/lib64/llvm14/ \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_EXE_LINKER_FLAGS=\"\$CMAKE_EXE_LINKER_FLAGS\" && \
+    make -j${BUILD_CORES} install && \
+    ln -s /usr/lib64/llvm14/lib/include/lld /usr/include/lld && \
+    cp /usr/lib64/llvm14/lib/liblld*.a /usr/local/lib/ && \
+    cd /tmp && rm -rf lld-* libunwind-*"
+
+# Build and install WasmEdge
```

Comment:
https://github.com/tequdev/xahaud/pull/4

This would have satisfied my curiousity before